nvflare/private/fed/client/client_run_manager.py

```diff
@@ -115,6 +115,9 @@ def __init__(
 
         self.logger = logging.getLogger(self.__class__.__name__)
 
+    def send_job_heartbeat(self, fl_ctx: FLContext):
```

Comment:
Why do we need this? No one will call this method.

Reply:
Removed.